overlord/fdestate/fdestate.go

```diff
@@ -218,7 +219,12 @@ func initializeState(st *state.State) error {
 
 	// FIXME mount points will be different in recovery or factory-reset modes
 	// either inspect degraded.json, or use boot.HostUbuntuDataForMode()
-	dataUUID, dataErr := disksDMCryptUUIDFromMountPoint(dirs.SnapdStateDir(dirs.GlobalRootDir))
+	dataDir := dirs.GlobalRootDir
```

Comment:
isn't this true on hybrid as well? Unless, I'm misunderstanding what the problem is and which path exists in which scenario.
```suggestion
	dataDir := dirs.SnapdStateDir(dirs.GlobalRootDir)
```

Reply:
I do not think it is a mount point on hybrid. On core it is a bind mount.